src/components/input-field/input-field.tsx

```diff
@@ -827,32 +823,23 @@ export class InputField {
                     }}
                     onDismiss={this.handleCloseMenu}
                 >
-                    {content}
+                    {this.renderListResult()}
                 </limel-menu-surface>
             </limel-portal>
         );
     };
 
-    private renderDropdown = () => {
-        const content = this.renderListResult();
-
-        return this.renderPortal(content);
-    };
-
     private renderListResult = () => {
-        const filteredCompletions: ListItem[] = this.filterCompletions(
-            this.value
-        );
-        if (!filteredCompletions || filteredCompletions.length === 0) {
-            return null;
+        if (!this.filterCompletions(this.value)) {
```

Comment:
Do we want to show an empty `limel-list` when the filtered completions are empty? This seems like an unintentional change, not refactoring.

Reply:
This doesn't return an empty limel-list. This prevents the limel-list from being rendered, if the search in auto-completions is not finding anything (both the old and the refactored code). 

However, they are both bad anyway. Because we should basically not render the portal at all, if:
1. there are no auto complete items
2. there are no filtered items (search results) 
3. And I think the check for `if (this.type === 'textarea' ` then return is also completely unnecessary.

 
I don't like this original code at all. 
I would have written it in a different way.

However, what is bad now, will be good for my next PR. 
I want to render the limel-helper-line in the portal. So these bad choices will be useful later. ;P